target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-wifi.dts

```diff
@@ -0,0 +1,402 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Copyright (c) 2016, 2018 The Linux Foundation. All rights reserved.
+ * Copyright (c) 2016 Google, Inc
+ */
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	model = "Google WiFi (Gale)";
+	compatible = "google,wifi", "google,gale-v2", "qcom,ipq4019";
```

Comment:
Which one of these is needed?
The SoC one can be dropped for sure.

Reply:
"google,wifi" is needed by OpenWRT for the naming scheme (matching this name with the board definitions in the Makefile), as discussed a ways back: http://patchwork.ozlabs.org/project/openwrt/patch/20200718205148.1743807-6-computersforpeace@gmail.com/.

"google,gale-v2" is needed by the bootloader, as it's looking for this product (code) name.

"qcom,ipq4019" is documented here: https://www.kernel.org/doc/Documentation/devicetree/bindings/arm/qcom.yaml

Why would I drop any of these?

Reply:
You don't need the SoC compatible at all, take a look at other boards.